lab-examples/clos02/setup.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+CFG_DIR=./configs
+
+configure_SRL () {
+  OUT=$(gnmic -a clab-clos02-$1 --timeout 30s -u admin -p admin -e json_ietf --skip-verify set --update-path / --update-file $CFG_DIR/$1.yaml 2>&1)
+  echo $OUT | grep -q -e '\"operation\": \"UPDATE\"'
+  if [ $? -eq 0 ]; then
+    SAVE=$(docker exec -it clab-clos02-$1 sr_cli "save startup" 2>&1)
+  else
+    echo "Error: Unable to push config. Check if NE is reachable with gRPC enabled."
+  fi
+}
+
+if [ $# -eq 0 ]; then
+  echo "Error: No topology file passed."
+else
+  DEPLOY=$(containerlab deploy --topo $1 2>&1)
```

Comment:
let's not hide the deployment process
instead, call the deploy command directly, so that its output is visible as the labs undergoes the deployment process
then you just check that exit status and exit if it is not `0`

Reply:
Removed deploy from setup. Rather kept it as a manual execution.